toxcore/group_chats.c

```diff
@@ -4744,7 +4744,7 @@ int gc_founder_set_max_peers(GC_Chat *chat, uint16_t max_peers)
     return 0;
 }
 
-int gc_send_message(const GC_Chat *chat, const uint8_t *message, uint16_t length, uint8_t type)
+int gc_send_message(const GC_Chat *chat, const uint8_t *message, uint16_t length, uint8_t type, uint32_t *pseudo_msg_id_param)
```

Comment:
Strange variable name. I think `message_id` would be better.

Reply:
done